src/Internal/Plutip/Server.purs

```diff
@@ -376,6 +383,7 @@ startPostgresServer pgConfig params = do
     databaseDir = workingDir <> "/postgres/data"
     postgresSocket = workingDir <> "/postgres"
     testClusterDir = (dirname <<< dirname) params.nodeConfigPath
+  sig <- liftEffect $ cleanupOnSigint workingDir testClusterDir
```

Comment:
The onSIGINT listener cleans up the directory as it should, except if the signal is sent between the creation of the directory (done on line 379) and the registering of the listener. but I guess this problem can't be solved, as the listener can only be registered once the directory name is known?